### DISCUSSION TITLE
Is this allow run in IOS store ?  (no JIT is allowd)

### DISCUSSION THREAD

**Discussion by calvin2021y** [WasmEdge/WasmEdge#1562] [Q&A]
Is this allow run in IOS store ?

**Comment by juntao** [comment]
I think it should be okay. After all, the iOS WebView itself has an embedded Wasm runtime.

And this Amazon story mentioned that they embedded a Wasm runtime in an app for over 8,000 different device types -- it did not explicitly say iOS, however. 

https://www.amazon.science/blog/how-prime-video-updates-its-app-for-more-than-8-000-device-types

**Comment by calvin2021y** [comment]
iOS WebView is part of system, you can run wasm inside jscore or webview beacause they are provide by system.

No IOS store app is allow to run JIT code or download code.  (all exe section must being in app package and not download from internet.)

**Reply by juntao** [reply]
Then just bundle Wasm in your app and do not download it from the internet ...
